packages/web-components/fast-foundation/src/tabs/index.ts

```diff
@@ -1,4 +1,4 @@
 export * from "./tabs.template";
 export * from "./tabs";
-export * from "./tab/index";
-export * from "./tab-panel/index";
+export * from "../tab/index";
+export * from "../tab-panel/index";
```

Comment:
should these be exported from their own index file rather than this one now that we've gone to a flat structure?